README.md

```diff
@@ -67,7 +67,7 @@ animation_format = "webp"
 The `[sticker]` section ist optional and can be left out.
 
 `transparent_color` is used as color for semitransparent pixel in `gif`s.
-The field has no effect, if the sticker is not animated or will be converted to `webp` (default).
+The field has no effect, if the sticker is not animated or will be converted to `webp` (default) or gif.
```

Comment:
explain more

Reply:
transparent_color is only used for gifs. webp has always transparency, including semi transparent pixel.

Reply:
Yes I saw it and still confused, this is why I added my notes to it :joy:

and got it. I added gif to be one of options for converted pictures, did not mean to link it with transparent image option

Reply:
Yeah this could probably be replaced with _This field has no effect unless the stickerpack contains animated stickers and those stickers are converted to gif_. Possibly also with and expanded explanation, that both png and webp support a full alpha channel, while gif only has opacity 0 and 1, meaning that pixels with an opacity between the two extremes ("semitransparent pixels") will use the provided transparent color to "resolve" the opacity.

The way you changed the sentence however implies that the transparent color, used for gifs, has no effect for gifs. That kinda contradicts itself.

Reply:
Yeah, I did not notice I added or gif in transparent_color. updated it https://github.com/LuckyTurtleDev/mstickereditor/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R73